protected/widgets/menu/SideMenu.php

```diff
@@ -53,9 +57,14 @@ public function init()
         ];
 
         Html::addCssClass($options, 'menu');
+        if($this->foldable) Html::addCssClass($options, 'foldable');
```

Comment:
Don't use inline ifs, they reduce readability. (https://www.php-fig.org/psr/psr-2/)

```suggestion
    if ($this->foldable) {
        Html::addCssClass($options, 'foldable');
    }
```